<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Arjun" ignored="0" name="Test" tests="9" failures="2" timestamp="03 Jul 2018 07:02:20 GMT" time="0.177" errors="0">
  <testcase name="testt1" time="0.027" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt1 -->
  <testcase name="testt1" time="0.002" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt1 -->
  <testcase name="testt1" time="0.003" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt1 -->
  <testcase name="testt1" time="0.002" classname="Tests.myTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.myTest.testt1(myTest.java:9)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testt1 -->
  <testcase name="testt2" time="0.006" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt2 -->
  <testcase name="testt2" time="0.01" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt2 -->
  <testcase name="testt2" time="0.002" classname="Tests.myTest">
    <skipped/>
  </testcase> <!-- testt2 -->
  <testcase name="testt2" time="0.002" classname="Tests.myTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.myTest.testt2(myTest.java:14)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testt2 -->
  <testcase name="testt3" time="0.001" classname="Tests.myTest"/>
</testsuite> <!-- Test -->
